src/os/ui/abstractmainctrl.js

```diff
@@ -259,6 +260,9 @@ os.ui.AbstractMainCtrl.prototype.initialize = function() {
   // set up trusted URI's
   os.net.loadTrustedUris();
 
+  // initialize variable replacers for time values in URI's
+  os.time.replacers.init();
```

Comment:
Hopefully any extending applications are still calling this `initialize` function, but in case they fully override it, this could cause some annoying breaks for them.

Reply:
All apps I'm aware of extend this base main controller, but if they don't they'll lose these URL replacers. We can make importing the file add the replacers because the circular dependency was related to the file that required it. Adding the require somewhere else would still work, but in general I don't like the idea of changing app behavior on import alone. An unintentional import could have unwanted side effects to an application.